backend/app/api/routes/dde.py

```diff
@@ -426,52 +433,23 @@ async def extract_from_file(
             status_code=status.HTTP_404_NOT_FOUND,
             detail="DocumentDataExtractor has no owner",
         )
-    # Build examples
-    examples = tup(
-        *(
-            tup(
-                as_text(
-                    document_data_extractor.owner,
-                    example.document_id,
-                    example.start_page,
-                    example.end_page,
-                ),
-                example.data,
-            )
-            for example in document_data_extractor.document_data_examples
-        )
-    )
-    # Pull data from the file
-    if upload.content_type != "application/pdf":
+    
+    try:
+        upload_content_type= ContentType(upload.content_type)     
+    except:
```

Comment:
bare excepts are bad practices: you should catch the correct exception being `ValueError`